src/components/articlecard/ArticleCard.svelte

```diff
@@ -156,7 +156,19 @@
               {/if}
             </div>
           {/if}
-          <h2 class="card-title {truncateTitle ? 'card-title--truncated' : ''}" style={titleStyle}>{title}</h2>
+          {#if videoIcon && !media}
+            <div class="video-icon">
+              <Icon className="color--white" name="videographic" width="20" />
+            </div>
+          {/if}
+          <h2
+            class="card-title {truncateTitle ? 'card-title--truncated' : ''} {videoIcon && !media
+              ? 'padding-l--r'
+              : ''}"
```

Comment:
det er rimelig ulæseligt synes jeg, kunne det ikke blive en variabel i stedet?

Reply:
Det var nu også på padding-l--r

Reply:
kunne man ikke gøre noget lignende med videoicon? 
det virker lidt sjovt at det kun er den ene der er ændret

Reply:
Jo selvfølgelig, det gik lidt for hurtigt - incoming!